modules/statics/src/networks.ts

```diff
@@ -294,7 +295,9 @@ class Eos extends Mainnet implements AccountNetwork {
 
 class EosTestnet extends Testnet implements AccountNetwork {
   family = CoinFamily.EOS;
-  explorerUrl = 'https://jungle.bloks.io/transaction/';
+  explorerUrl = 'https://local.bloks.io/transaction/';
+  urlParams =
```

Comment:
do you still need this given https://github.com/BitGo/bitgo-microservices/pull/10947/files#diff-f6421b8a50f8aff713d166802c0070055b0f48f6deaecbbc50603d8440638ebbR441